.vscode/settings.json

```diff
@@ -96,5 +96,11 @@
   "cSpell.words": ["Discoverability"],
   "typescript.tsdk": "common/temp/node_modules/typescript/lib",
   "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "files.insertFinalNewline": true
+  "files.insertFinalNewline": true,
+  "eslint.experimental.useFlatConfig": true,
+  "eslint.workingDirectories": [
```

Comment:
What is the workingDirectories setting here for?

Reply:
VSCode ESLint extension does not function properly without it. If removed, the ESLint server spits out this error:
![image](https://github.com/iTwin/appui/assets/98850418/dff21833-060d-46c9-86ef-99e2b8913c0c)

Reply:
Is this still needed after the change to the location of the base config?